ncd-calculator/src/functions/log.js

```diff
@@ -0,0 +1,72 @@
+import pino from 'pino';
+
```

Comment:
if we decide to log to a file from front end calls, add a destination to log on
i.e front-end.log

then add rolling files based on date changes / or fize size when they are too heavy (> 50 mb) or something
then mount that folder to the container

imo only log important things because frontend calls will be a lot